examples/src/main/java/org/apache/spark/examples/ml/JavaTokenizerExample.java

```diff
@@ -23,17 +23,23 @@
 import java.util.Arrays;
 import java.util.List;
 
+import scala.collection.mutable.WrappedArray;
+
 import org.apache.spark.ml.feature.RegexTokenizer;
 import org.apache.spark.ml.feature.Tokenizer;
+import org.apache.spark.sql.api.java.UDF1;
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.RowFactory;
-import org.apache.spark.sql.types.DataTypes;
-import org.apache.spark.sql.types.Metadata;
-import org.apache.spark.sql.types.StructField;
-import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.types.*;
```

Comment:
Here imports are collapsed to *; elsewhere a * import is expanded. I might generally not touch these, but, the standard is usually to avoid wildcard imports by default